alignak_backend/models/servicegroup.py

```diff
@@ -131,6 +131,8 @@ def get_schema():
                     'resource': 'servicegroup',
                     'embeddable': True
                 },
+                'nullable': True,
+                'default': None
```

Comment:
If we have `none` per default, why permit be nullable ?

Reply:
Setting only default=None is not accepted, nullable must then be specified